api/v1alpha1/mirrorpeer_types.go

```diff
@@ -45,12 +54,22 @@ type MirrorPeerSpec struct {
 	Items []PeerRef `json:"items"`
 }
 
+type SecretExchangeStatus struct {
+	UpdateTimestamp metav1.Time `json:"updateTimestamp"`
+	Message         string      `json:"message,omitempty"`
+	Reason          string      `json:"reason,omitempty"`
+}
+
 // MirrorPeerStatus defines the observed state of MirrorPeer
 type MirrorPeerStatus struct {
+	AllClustersOnline               bool                   `json:"clusterOnline,omitempty"`
+	PeeringStatus                   PeeringStatusType      `json:"peeringStatus,omitempty"`
```

Comment:
I don't think we have a way to determine this.

Reply:
The peering status can be assumed with the presence of addon, clusters and secrets sync right ? Or should we remove it